src/ExtensionsSample/Samples/DocumentDBSamples.cs

```diff
@@ -45,7 +51,7 @@ public static class DocumentDBSamples
         //
         // This example uses the binding template "{QueueTrigger}" to specify that the Id should come from
         // the string value of the queued item.
-
+        [Disable]
```

Comment:
Yup -- in general, ignore this file. I meant to exclude it from the commit but obviously missed it.

---
In reply to: [63893331](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/97#discussion_r63893331) [](ancestors = 63893331)